apps/mobile/src/features/ConfirmTx/components/SignTransaction/SignError.tsx

```diff
@@ -0,0 +1,67 @@
+import { StyleSheet } from 'react-native'
+import { LinearGradient } from 'expo-linear-gradient'
+import React from 'react'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { ScrollView, Text, View } from 'tamagui'
+import { Badge } from '@/src/components/Badge'
+import { SafeFontIcon } from '@/src/components/SafeFontIcon'
+import { LargeHeaderTitle } from '@/src/components/Title'
+import { SafeButton } from '@/src/components/SafeButton'
+import { router } from 'expo-router'
+
+const colors: [string, string] = ['#221818', 'transparent']
```

Comment:
Is this color not in the theme?

Reply:
it's not, because the error colors are different.

Reply:
Just checked how it looks with the theme colors, and looks ok as well, will use the theme ones and ask @TanyaEfremova to update Figma screens